Lib/asyncio/tools.py

```diff
@@ -246,6 +268,20 @@ def display_awaited_by_tasks_table(pid: int) -> None:
         print(f"{row[0]:<10} {row[1]:<20} {row[2]:<20} {row[3]:<50} {row[4]:<50} {row[5]:<15} {row[6]:<15}")
 
 
+def _display_awaited_by_tasks_csv(table, format: TaskTableOutputFormat) -> None:
+    _header = ('tid', 'task id', 'task name', 'coroutine stack', 'awaiter chain', 'awaiter name', 'awaiter id')
```

Comment:
The standard library adheres to PEP 8. Please keep line lengths in check:

```suggestion
    csv_headers = ('tid', 'task id', 'task name', 'coroutine stack',
                   'awaiter chain', 'awaiter name', 'awaiter id')
```